source/user-manual/reference/ossec-conf/integration.rst

```diff
@@ -151,12 +154,15 @@ Configuration example
     <level>10</level>
     <group>multiple_drops,authentication_failures</group>
     <alert_format>json</alert_format>
+    <options>{"pretext": "Custom Title"}</options> <!-- Replace with your custom JSON object -->
   </integration>
 
   <!-- Integration with PagerDuty -->
   <integration>
     <name>pagerduty</name>
     <api_key>API_KEY</api_key> <!-- Replace with your PagerDuty API key -->
+    <options>{"pretext": "Custom title"}</options> <!-- Replace with your custom JSON object -->
+    <alert_format>json</alert_format> <!-- With the new script this is mandatory -->
```

Comment:
What new script? maybe some context is needed if commenting like this

Reply:
Hi @javimed !

I spoke with @MarcelKemp  and the new script is being maintained.

Mainly, the change is based on the fact that pagerduty previously had a `shell` script and now it is in `python`.
For this reason, as most of the python scripts of integrations, the format of the alert that it expects is in JSON and for this reason it is necessary to change this parameter in the configuration.

Here I let you the associate (PR)[https://github.com/wazuh/wazuh/pull/16090] and (issue)[https://github.com/wazuh/wazuh/issues/5833] 😃